includes/definitions/mobileiron.yaml

```diff
@@ -0,0 +1,24 @@
+os: mobileiron
+type: appliance
+group: unix
+text: MobileIron
+ifXmcbc: true
+ifname: true
+mib_dir:
```

Comment:
You might not need these other mib_dirs.  (but you need to test that first)

Reply:
The vendor allows you to deploy the appliance to your hardware if desired, so it'd be worth leaving those in just in case anyone does that.